test/baselines/adapter_JS_ES6Class_ESLint_TypeChecking_Spaces_SingleQuotes_LGPLv3/.create-adapter.json

```diff
@@ -0,0 +1,35 @@
+    },
```

Comment:
I think that's a bit too much unnecessary information. Can you check if there's a simple way to just store the license id? I remember that the licenses were handled a bit special.

Reply:
Yes, unfortunately licenses are changed from an index (number) to the license information in the answer structure. This would mean I have to figure out the index and then store that - and when replaying I have to convert again the index to the license information.
Another (preferred) solution would be to remove the entire license "transformation" from the questions and move it to where the license is handled - instead of an index we could still use the id.

Reply:
Its not a number, its the license key that is displayed - but that is different from the id.

> Another (preferred) solution would be to remove the entire license "transformation" from the questions and move it to where the license is handled - instead of an index we could still use the id.

I did the transform because I wouldn't have to do it in multiple templates this way. We can do what you proposed or use a reverse lookup in the answers file template:
```
import { entries } from "alcalzone-shared/objects";
const key = entries(licenses).find(([, license]) => license.id === id)?.[0];
```
where `licenses` is the export from `licenses.ts`, `id` is the wanted id.

Reply:
Changed to only store the name of the license. Everything is now done from that string. And added a helper function to generate the whole license text.